src/main/kotlin/racingcar/RandomNumberGenerator.kt

```diff
@@ -0,0 +1,10 @@
+package racingcar
+
+import camp.nextstep.edu.missionutils.Randoms
+
+class RandomNumberGenerator : NumberGenerator {
+
+    override fun generate(numberMin: Int, numberMax: Int): Int {
+        return Randoms.pickNumberInRange(numberMin, numberMax)
+    }
```

Comment:
```suggestion
object RandomNumberGenerator : NumberGenerator {

    override fun generate(numberMin: Int, numberMax: Int): Int {
        return Randoms.pickNumberInRange(numberMin, numberMax)
    }
```

What's the difference? Which one would be more suitable?